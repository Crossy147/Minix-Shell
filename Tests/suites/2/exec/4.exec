
$TESTED_SHELL < $inf 2> $errf | sort > $outf

