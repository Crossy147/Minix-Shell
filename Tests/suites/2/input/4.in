# builtin lls
lls
echo chdir to servers
lcd servers
lls
echo chdir to pm
lls
echo done
