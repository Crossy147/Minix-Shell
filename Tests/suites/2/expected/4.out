Makefile
Makefile
bin
chdir to pm
chdir to servers
done
fs
fs
init
init
pm
pm
rs
rs
servers
